src/stubs/ErrorUtils.js

```diff
@@ -11,13 +11,17 @@
 
 /* jslint unused:false */
 
-var ErrorUtils = {
-  applyWithGuard(callback, context, args, onError, name) {
-    return callback.apply(context, args);
-  },
-  guard(callback, name) {
-    return callback;
-  },
-};
+if (global.ErrorUtils) {
```

Comment:
Thanks for checking! We should perhaps move this into https://github.com/facebook/fbjs/tree/master/src/__forks__ then (since stubs/ is actually a thing in s_u and that's where EventListener lives). Doesn't really matter for anybody except a couple people like me and you who actually mostly know this stuff :P.